src/libraries/Native/AnyOS/LdapProxy.Native/entrypoints.c

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "../entrypoints.h"
```

Comment:
This file should not be needed. It is only needed for shims that are part of netcoreapp.

Reply:
yeah, I just wanted to ask about it, `*ResolveDllImport` methods are not used in Runtime repo, should I delete only this one or all entrypoints.c are dead?

Reply:
Just this one.

`*ResolveDllImport` methods are used for single file build.